ingestion/src/metadata/ingestion/source/bigquery_usage.py

```diff
@@ -130,16 +136,85 @@ def next_record(self) -> Iterable[TableQuery]:
                                 service_name=self.config.service_name,
                             )
                             yield tq
+
+                            result = LineageRunner(tq.sql)
+                            metadata = OpenMetadata(self.metadata_config)
+                            if result.target_tables and result.intermediate_tables:
+                                for intermediate_table in result.intermediate_tables:
+                                    intermediate_fqdn = f"{self.config.service_name}.{intermediate_table}"
```

Comment:
Does the intermediate_table contains the database? FQDN should be service_name.database.table